public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -19,6 +19,8 @@ interface State {
 
 export class DashboardLinksDashboard extends PureComponent<Props, State> {
   state: State = { resolvedLinks: [] };
+  wrapperRef: RefObject<HTMLDivElement> = React.createRef();
+  listItemRef: RefObject<HTMLUListElement> = React.createRef();
```

Comment:
nitpick: maybe you can add `createRef` to imports and then specify the type?
```suggestion
  wrapperRef = createRef<HTMLDivElement>();
  listItemRef = createRef<HTMLUListElement>();
```